<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elemet_Timer</class>
 <widget class="QDialog" name="Elemet_Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="horizontalSpacing">
    <number>25</number>
   </property>
   <property name="verticalSpacing">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <item row="0" column="0">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>H:mm:ss AP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Count Repeat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="TimesToRepeat"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="set_time">
     <property name="text">
      <string>Add Timer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="NameEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
